mobify/progressive-web-scaffold
# Scaffold renaming

**JIRA**: (link to JIRA ticket)
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/512

## Changes
- Rename `raw` to `unwrapped` in `templates.js`
- `app-provider.js` to `router.js`
- add updated `npm link` instructions

## How to test-drive this PR
- Do an `npm link` to related PR (see instructions IN the README thanks to this PR and @jeremywiebe )
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
